specification/web/resource-manager/Microsoft.Web/stable/2024-11-01/CommonDefinitions.json

```diff
@@ -673,6 +673,311 @@
         }
       }
     },
+    "Certificate": {
+      "description": "SSL certificate for an app.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Certificate resource specific properties",
+          "type": "object",
+          "properties": {
+            "password": {
```

Comment:
>password [](http://example.com/codeflow?start=13&length=8)

mark it as a secret using the x-ms-secret attribute

Reply:
Done